docs/querying/filters.md

```diff
@@ -531,3 +531,16 @@ The true filter is a filter which matches all values.  It can be used to tempora
 
 { "type" : "true" }
 ```
+
+### Expression Filter
+The expression filter allows for the implementation of arbitrary conditions, leveraging the Druid expression system. This filter might be less performant than a combination of the other filters on this page due to the fact that not all filter optimizations are in place yet.
```

Comment:
```suggestion
The expression filter allows for the implementation of arbitrary conditions, leveraging the Druid expression system. 

This filter allows for more flexibility, but it might be less performant than a combination of the other filters on this page due to the fact that not all filter optimizations are in place yet.
```